core/src/datayoga_core/blocks/relational/write/block.py

```diff
@@ -59,6 +59,25 @@ def init(self, context: Optional[Context] = None):
 
     async def run(self, data: List[Dict[str, Any]]) -> BlockResult:
         logger.debug(f"Running {self.get_block_name()}")
+
+        try:
+            # try to process all records together
+            return await self._run(data)
+        except Exception as e:
```

Comment:
we need to make sure that things like retry connection are still handled the same way. Not sure how this will work if we catch all exceptions? how is this handled today?